src/main/java/com/powsybl/openloadflow/dc/equations/DcEquationSystem.java

```diff
@@ -51,19 +51,24 @@ public static void createNonImpedantBranch(EquationSystem<DcVariableType, DcEqua
             equationSystem.createEquation(branch.getNum(), DcEquationType.ZERO_PHI)
                     .addTerm(equationSystem.getVariable(bus1.getNum(), DcVariableType.BUS_PHI).createTerm())
                     .addTerm(equationSystem.getVariable(bus2.getNum(), DcVariableType.BUS_PHI).<DcEquationType>createTerm()
-                                         .minus());
+                                         .minus()).setActive(!branch.isDisabled());
```

Comment:
I really don't understand why but this first ``setActive`` is not needed in AcEquationSystem. So I don't know if this fix is the good one.

Reply:
EquationSystem.createEquation has 2 overloads:
 - initial one just take an element num
 - newest one take directly the LfElement and so on is able to internally disable equation according to LfElement status

DcEquationSystem had unfortunately not been migrated to new one